src/frontends/pytorch/src/op/embedding_bag.cpp

```diff
@@ -48,15 +53,17 @@ OutputVector translate_embedding_bag_common(const NodeContext& context) {
         PYTORCH_OP_CONVERSION_CHECK(!include_last_offset, "Inclusion last offset is not supported");
         // no per_sample_wights
         if (context.input_is_none(6)) {
-            result = context.mark_node(std::make_shared<ov::op::v3::EmbeddingBagOffsetsSum>(weight, indices, offsets));
+            result = context.mark_node(
+                std::make_shared<ov::op::v15::EmbeddingBagOffsets>(weight, indices, offsets, reduction));
         } else {
             auto per_sample_weight = context.get_input(6);
             per_sample_weight = context.mark_node(std::make_shared<ov::op::v1::ConvertLike>(per_sample_weight, weight));
-            result = context.mark_node(std::make_shared<ov::op::v3::EmbeddingBagOffsetsSum>(weight,
-                                                                                            indices,
-                                                                                            offsets,
-                                                                                            zero,
-                                                                                            per_sample_weight));
+            result = context.mark_node(std::make_shared<ov::op::v15::EmbeddingBagOffsets>(weight,
+                                                                                          indices,
+                                                                                          offsets,
+                                                                                          zero,
+                                                                                          per_sample_weight,
+                                                                                          reduction));
```

Comment:
As the op version produced by Pytorch FE is updated for both modes, downgrade transformation should be added to ensure backward compatibility for other plugins like GPU with `v3::EmbeddingBagOffsetsSum`/`v3::EmbeddingBagPackedSum` support only https://github.com/openvinotoolkit/openvino/blob/350c021870309ecdc00eba4ceef64b2029cfe9f8/src/plugins/intel_gpu/src/plugin/ops/embedding_bag.cpp#L20

To ensure backward compatibility without downgrade transformation, the v15 version could be produced only for the `MEAN` mode.

Reply:
Done, enabled transformation globally for all plugins. For CPU implementation is same for both v15 and v3 sum mode, so it shouldn't change anything except used op in IR